backend/index.js

```diff
@@ -6,8 +6,10 @@ const app = express()
 
 app.use('/api/v1', cors(), router)
 
-const port = 8001
+// const port = 8001
 
+/*
 app.listen(port, () =>
   console.log(`backend service listening at port ${port}`)
 )
+*/
```

Comment:
I don't get why comment this out.
All these lines are very important.

Reply:
Do you want me to put this log as a debug too? The log throws a warning on the lint